sippy-ng/src/component_readiness/CompReadyUtils.js

```diff
@@ -79,34 +79,39 @@ export const cancelledDataTable = {
 export const jiraUrlPrefix = 'https://issues.redhat.com/browse/'
 
 // Make one place to create the Component Readiness api call
-export function getAPIUrl() {
-  return process.env.REACT_APP_API_URL + '/api/component_readiness'
+export function getAPIUrl(endpoint) {
+  return `${process.env.REACT_APP_API_URL}/api/${endpoint}`
 }
 
-export function getJobVariantsUrl() {
-  return process.env.REACT_APP_API_URL + '/api/job_variants'
+export function getCRMainAPIUrl() {
+  return getAPIUrl('component_readiness')
 }
 
-export function getComponentReadinessViewsUrl() {
-  return process.env.REACT_APP_API_URL + '/api/component_readiness/views'
+export function getJobVariantsAPIUrl() {
+  return getAPIUrl('job_variants')
+}
+
+export function getComponentReadinessViewsAPIUrl() {
+  return getAPIUrl('component_readiness/views')
 }
 
 // Make one place to create the Component Readiness test_details api call
 export function getTestDetailsAPIUrl() {
-  return process.env.REACT_APP_API_URL + '/api/component_readiness/test_details'
+  return getAPIUrl('component_readiness/test_details')
 }
 
-export function getTriagesAPIUrl() {
-  return process.env.REACT_APP_API_URL + '/api/component_readiness/triages'
+export function getTriagesAPIUrl(id) {
+  return getAPIUrl(
+    id ? `component_readiness/triages/${id}` : 'component_readiness/triages'
+  )
```

Comment:
I'm not sure I like this change. It isn't clear, without understanding the internals of the function, that the `id` arg can be omitted. Maybe we could have a separate function that doesn't take the arg?

Reply:
I couldn't include the type indication since i don't really understand how my IDE/linter treat typescript vs javascript but at least including the `null` default is pretty clear.